test/controllers/resource_controller_test.rb

```diff
@@ -0,0 +1,30 @@
+require "test_helper"
+
+class ResourceControllerTest < ActionDispatch::IntegrationTest
```

Comment:
@Pralish can we add a test case that simulates a more complex form submission? Like including a file upload and rich text blob